Content.Shared/Drunk/DrunkSystem.cs

```diff
@@ -9,8 +11,33 @@ public abstract class SharedDrunkSystem : EntitySystem
     [ValidatePrototypeId<StatusEffectPrototype>]
     public const string DrunkKey = "Drunk";
 
-    [Dependency] private readonly StatusEffectsSystem _statusEffectsSystem = default!;
+    [ValidatePrototypeId<StatusEffectPrototype>]
+    public const string SleepKey = "ForcedSleep";
+    
+    [Dependency] protected readonly StatusEffectsSystem StatusEffectsSystem = default!;
```

Comment:
Why is it protected?

Reply:
is is used by server DrunkSystem, which inherits from Shared